pkgs/servers/pr-tracker/default.nix

```diff
@@ -0,0 +1,35 @@
+{ rustPlatform
+, lib
+, fetchgit
+, openssl
+, pkg-config
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pr-tracker";
+  version = "1.0.0";
+
+  src = fetchgit {
+    url = "https://git.qyliss.net/pr-tracker";
+    rev = version;
+    sha256 = "sha256-NHtY05Llrvfvcb3uyagLd6kaVW630TIP3IreFrY3wl0=";
+  };
+
+  cargoSha256 = "sha256-SgSASfIanADV31pVy+VIwozTLxq7P3oMDIiAAQ8s+k0=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl systemd ];
+
+  meta = with lib; {
+    description = "Nixpkgs pull request channel tracker";
+    longDescription = ''
+      A web server that displays the path a Nixpkgs pull request will take
+      through the various release channels.
+    '';
+    platforms = platforms.linux;
+    homepage = "https://git.qyliss.net/pr-tracker";
+    license = licenses.mit;
```

Comment:
Project isn't licensed under the MIT, but instead under the GNU Affero General Public License

Reply:
:facepalm: that's what I get from copying from another package. Should be fixed now.